krcal/map_builder/map_builder_functions_test.py

```diff
@@ -90,6 +92,38 @@ def test_maps_nans_outside_rmax(xy_pos, output_maps_tmdir):
     assert all(np.isfinite(coefs[maskin ]))
 
 
+def test_detector_dependence(output_maps_tmdir,
+                             folder_test_dst  ,
+                             test_dst_file    ):
+    previous_map_f = os.path.join(output_maps_tmdir, 'test_out_map.h5')
+    previous_map = read_maps(previous_map_f)
+
+    config = configure('maps $ICARO/krcal/map_builder/config_LBphys.conf'.split())
+    histo_file_out   = os.path.join(output_maps_tmdir, 'test_out_histo.h5')
+    demo_map_file_out= os.path.join(output_maps_tmdir, 'test_out_map_demo.h5')
+    default_n_bins   = 15
+    run_number       = 7517
+    map_params_new   = copy.copy(config.as_namespace.map_params)
+    map_params_new['nmin']     = 100
+    map_params_new['detector'] = 'demopp'
+    config.update(dict(folder         = folder_test_dst  ,
+                       file_in        = test_dst_file    ,
+                       file_out_map   = demo_map_file_out,
+                       file_out_hists = histo_file_out   ,
+                       default_n_bins = default_n_bins   ,
+                       run_number     = run_number       ,
+                       map_params     = map_params_new   ))
+
+    map_builder(config.as_namespace)
+    demo_map = read_maps(demo_map_file_out)
+    previous_driftv = previous_map.t_evol.dv
+    demo_driftv     = demo_map    .t_evol.dv
+    z_cathode_new  = DB.DetectorGeo('new').ZMAX[0]
+    z_cathode_demo = DB.DetectorGeo('demopp').ZMAX[0]
+    ratio_cathodes = z_cathode_new/z_cathode_demo
+    assert_allclose ((previous_driftv/demo_driftv).values, ratio_cathodes)
+
+
```

Comment:
My problem with this test is that it relays on the fact that, for this particular set of runs, the drift velocity is the same. So, might work as a particular numerical comparison, but is it right as a general one? Am I too picky?

Reply:
Please, @mmkekic ,  whenever you are available, could you take a quick look here?